bcipy/helpers/stimuli.py

```diff
@@ -31,30 +52,38 @@ class InquirySchedule(NamedTuple):
     colors: List[List[str]]
 
 
-def rsvp_inq_generator(query: list,
-                       timing=[1, 0.2],
-                       color=['red', 'white'],
-                       stim_number=1,
-                       is_txt=True,
-                       inq_constants=None) -> InquirySchedule:
+def alphabetize(stimuli: List[str]) -> List[str]:
+    """Alphabetize.
+
+    Given a list of string stimuli, return a list of sorted stimuli by alphabet.
+    """
+    return sorted(stimuli, key=lambda x: re.sub(r'[^a-zA-Z0-9 \n\.]', 'ZZ', x).lower())
+
+
+def rsvp_inq_generator(query: List[str],
+                       timing: List[float] = [1, 0.2],
+                       color: List[str] = ['red', 'white'],
+                       stim_number: int = 1,
+                       stim_order: StimuliOrder = StimuliOrder.RANDOM.value,
```

Comment:
Based on the default value and your usage below, it looks like `stim_order` should have type `str`. Otherwise we need to change the default value should to `StimuliOrder.RANDOM` and adjust the comparison. I'm okay with this either way. It's coming out of the parameters file as a string so that might be easier to deal with (although maybe we should discuss supporting enums and doing a cast). However, it might be easier to communicate expected values if it has the type `StimuliOrder`.

Reply:
Looks like this comment applies to the other functions in this file as well.